AI Guidelines/iOS/FlowController.md

```
@@ -0,0 +1,113 @@
+      guard let settingsFlow = flow as? SettingsFlow else { return }
+      switch settingsFlow {
+      case .notifications: presentNotifications()
+      case .about: presentAbout()
+      // ...
+      }
+  }
+  ```
+
+---
+
+## 7. **Navigation and Child Flows**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)